doc/README.md

```diff
@@ -514,15 +516,32 @@ Expected outcome
 * ARC responds with status `ANNOUNCED_TO_NETWORK` or `SEEN_ON_NETWORK`
 * At latest a couple of minutes later the status will switch to `SEEN_ON_NETWORK`
 
-### Block reorg
+### Chain reorg
 
-A block reorg happens and two different blocks get announced at the same block height
+A chain reorganization (chain reorg) occurs when the blockchain switches its primary chain of blocks, usually because a previously stale chain (fork) has grown to surpass the current longest chain. This process can affect transaction statuses based on their presence in the involved chains.
```

Comment:
We could state here that the deciding factor whether a reorg happens is by comparing the chainwork of a competing chain. What do you think?

Reply:
Added. Let me know if it's okay.

I can always describe it more if you think it's necessary.